src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.logo-container a {
+  display: block;
+}
+
+.logo-container img {
+  display: block;
+}
+
+.nav ul {
```

Comment:
do not style by tag (maybe just for reset default styles, body, html)
add class for ul
check same cases below